resources/js/beatmap-discussions/new-discussion.tsx

```diff
@@ -39,7 +40,7 @@ interface DiscussionsCache {
 
 interface Props {
   autoFocus: boolean;
-  beatmapset: BeatmapsetExtendedJson & BeatmapsetWithDiscussionsJson;
+  beatmapset: BeatmapsetExtendedJson & BeatmapsetWithDiscussionsLoggedInJson;
```

Comment:
This component is rendered when being guest so the typing isn't quite correct.

And if anything I'd rather have the `current_user_attributes` to be always returned instead of adding even more specialized type.